index.html

```diff
@@ -25,7 +25,7 @@
           </section>
         </aside>
       </div>
-      <div id = "legend-house">
+      <div id = "legend-house" data-city = "Lviv" data-street = "Staroyevreyska" data-number-street = "48" index = "79000">
```

Comment:
@Vovan707 `zip-code`, not `index`. Also, `data-` suffix is missed.

Do you review your changes yourself?